src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -76,12 +76,16 @@ def get_metadata(self) -> dict:
         # Device metadata
         device = get_device_metadata(self.meta)
 
+        # Should follow pattern 'Imec0', 'Imec1', etc.
+        probe_name = self.stream_id[:5].capitalize()
```

Comment:
Unfortunately could not think of a way this could be done purely through utils since the `.meta` dictionary doesn't seem to have much about the probe number (as it was plugged and indexed alongside other devices)

Reply:
We have the slot and the probe number and we can get it from there but I can do that in another PR. Plus, I discuss with Alessio and we want to propagate this to the probe metadata and then we should fetch it from there (spikeglx always has a probe loaded by default).